magnum_cluster_api/driver.py

```diff
@@ -186,6 +192,10 @@ def delete_cluster(self, context, cluster):
         resources.ClusterResourceSet(self.k8s_api, cluster).delete()
         resources.ClusterResourcesConfigMap(context, self.k8s_api, cluster).delete()
         resources.Cluster(context, self.k8s_api, cluster).delete()
+        if utils.get_cluster_label_as_bool(self.cluster, "auto_scaling_enabled", False):
+            resources.ClusterAutoscalerHelmRelease(self.k8s_api, cluster).delete()
+            resources.ClusterAutoscalerHelmRepository(self.k8s_api, cluster).delete()
```

Comment:
not a good idea, HelmRepository is used by all of the clusters, so it should always be there :)